fs/rc/internal.go

```diff
@@ -427,19 +442,30 @@ func rcRunCommand(ctx context.Context, in Params) (out Params, err error) {
 
 	cmd := exec.CommandContext(ctx, ex, allArgs...)
 
-	// Run the command and get the output for error and stdout combined.
-	output, err := cmd.CombinedOutput()
+	if returnType == "COMBINED_OUTPUT" {
+		// Run the command and get the output for error and stdout combined.
 
-	if err != nil {
-		fs.Errorf(nil, "Command error %v", err)
+		out, err := cmd.CombinedOutput()
+
+		if err != nil {
+			return Params{
+				"result": string(out),
+				"error":  true,
+			}, nil
+		}
 		return Params{
-			"result": string(output),
-			"error":  true,
+			"result": string(out),
+			"error":  false,
 		}, nil
+	} else if returnType == "STREAM_ONLY_STDOUT" {
+		cmd.Stdout = *httpResponse
```

Comment:
So this writes direct to the http output which is cool!

Do we need to change the Content-Type for this?

Reply:
I think we should set it to text/plain